content/en/serverless/azure_app_services.md

```diff
@@ -145,7 +223,7 @@ b. Ensure that any App Service Plan filtering rules you may have applied include
 a. Verify you've set `DD_SITE` and `DD_API_KEY` correctly.
 b. Do a full stop and start of your application.
 c. If not resolved, try uninstalling the extension and re-installing (this also ensures you are running the latest version).
-d. Still need help? Contact [Datadog support][18].
+d. Still need help? Contact [Datadog support][24].
```

Comment:
The formatting of this section is buggy, but I think it was before you added a letter to the list. Lines 223-226 should be indented to line up with the "I" in "If" of line 213.